plugins/woocommerce/bin/build-zip.sh

```diff
@@ -11,13 +11,12 @@ mkdir -p "$DEST_PATH"
 
 echo "Installing PHP and JS dependencies..."
 pnpm install
-pnpm nx composer-install woocommerce || exit "$?"
 echo "Running JS Build..."
-pnpm nx build woocommerce || exit "$?"
+pnpm exec turbo build --filter=woocommerce || exit "$?"
```

Comment:
I got this. 👇 

```bash
❯ pnpm exec turbo build --filter=woocommerce
Usage: turbo [--version] [--help] <command> [<args>]

Available commands are:
    bin       Get the path to the Turbo binary
    link      Link your local directory to a Vercel organization and enable remote caching.
    login     Login to your Vercel account
    logout    Logout of your Vercel account
    prune     Prepare a subset of your monorepo
    run       Run a task
    unlink    Unlink the current directory from your Vercel organization and disable Remote Caching (beta).
```

Should we use `pnpm exec turbo run build --filter=woocommerce ` instead?